pipeline/sources/authorities/lc/mapper.py

```diff
@@ -546,8 +546,11 @@ def transform(self, record, rectype=None, reference=False):
                     if txt:
                         if not bpid or bpid.startswith("_:"):
                             bpid = self.build_recs_and_reconcile(txt,"place")
-                        if bpid:
-                            where = model.Place(ident=bpid, label=txt)
+                    if bpid:
+                        #bpid is full uri
+                        check = bpid.rsplit("/",1)[-1]
+                        where = self.get_reference(check)
```

Comment:
Should use configs.split_uri(bpid) to get the source and the identifier. Then call source['mapper'].get_reference(identifier) ... otherwise it will try to get it from lcnaf ... which won't exist.